package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A bugzilla client for Activity Stream",
   "main": "main.js",
   "scripts": {
-    "prestart": "npm run generatelookups && npm run tsc:prod",
+    "prettier": "prettier --write \"src/**/*.{ts,tsx,js,jsx}\" --ignore-path .gitignore",
+    "prestart": "npm run prettier && npm run generatelookups && npm run tsc:prod",
```

Comment:
The workflow here seems like it could work well, since it doesn't add a side-effect to the lint step.